web/portal/brand/src/entities/Company/Form.tsx

```diff
@@ -104,7 +109,7 @@ const Form = (props: EntityFormProps): JSX.Element | null => {
     edit &&
       !isResidential && {
         legend: _('Platform data'),
-        fields: ['outgoingDdi', 'outgoingDdiRule'],
+        fields: [isVpbx && 'outgoingDdi', isVpbx && 'outgoingDdiRule'],
       },
```

Comment:
All the fields in this section are exclusive to vPBX clients, this can also be expressed like...

```
      isVpbx && {
        legend: _('Platform data'),
        fields: ['outgoingDdi', 'outgoingDdiRule'],
      },
```

Sorry not taking this into account at first, only do this change if you're willing to

Reply:
Done